packages/core/template/project/developerGuide/Testing.md

```diff
@@ -0,0 +1,31 @@
+---
+  layout: default.md
+  title: "Testing guide"
+  pageNav: 3
+---
+
+# Testing guide
+
+<box type="tip">
+This page can be used to understand the testing processes.
+</box>
+
+## Running tests
+
+* **Method 1: Using script**
+  * Create script for running tests in package.json
+  * Open a console and run the command for the script, like `npm run test`
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Types of tests
+
+This project has two types of tests:
+
+<panel header="**1. Unit tests**">
+<p>These tests are targeting the lowest level methods/classes.</p>
```

Comment:
```suggestion

These tests target the lowest level methods/classes.

Example command (replace with your own) `npm run test`
```

I think you shouldn't have to use html here? Can you check. I feel that using `<p>` tags is not really best / common markbind practie, though it's supported